srcs/pressure_controller.cpp

```diff
@@ -434,30 +449,30 @@ void PressureController::onPeakPressureIncrease(uint8_t p_increment) {
     }
 }
 
-void PressureController::updatePhase(uint16_t p_centiSec) {
-    if (p_centiSec < m_centiSecPerInhalation) {
+void PressureController::updatePhase(uint16_t p_tick) {
+    if (p_tick < m_tickPerInhalation) {
         m_phase = CyclePhases::INHALATION;
 
 #if VALVE_TYPE == VT_FAULHABER
-        uint16_t pressureToTest = m_maxPlateauPressureCommand;
+        uint16_t pressureToTest = m_maxPlateauPressureCommand-5;
```

Comment:
Can we add some explanation about the use of `m_maxPlateauPressureCommand - 5` here?

Reply:
done